source/protocol/lsp-server_notifications.ads

```diff
@@ -23,29 +23,8 @@ package LSP.Server_Notifications is
    type Server_Notification_Handler_Access is
      access all Server_Notification_Handler'Class;
 
```

Comment:
Can we keep clean and precise interface for server instead of replacing it with just one procedure? Why not to use separate dispatcher interface with `Handle_Notification` method. The dispatcher then calls distinct method of handler. All servers will not reimplement dispatcher, but provide custom handler.

Reply:
Agreed. Will do this for both Requests and Notifications.